tools/release.py

```diff
@@ -282,6 +283,11 @@ def commit_changlog_entries(*, version):
             )
 
 
+def escape_ansi(*, line):
+    ansi_escape = re.compile(r"(?:\x1B[@-_]|[\x80-\x9F])[0-?]*[ -/]*[@-~]")
```

Comment:
unsure about including UTF-8 directly here, is that a secure and best practices thing to do ?

Just remember issues of spoofing with \x80 which is a space and not valid to do, since they should have used \x20. This was recommended practise back at the end of the last century but don't know if recommendations have changed.

Reply:
Good catch, this isn't even used. So how this got merged in when I was resolving conflicts.